src/Tabs/detectScrollbarSize.js

```diff
@@ -1,36 +1,41 @@
-let horizontalScrollbarHeight
+let scrollbarSize
 const className = '__vertical-scrollbar-height-test__'
-const styles = `
+const baseCSS = `
     .${className} {
         position: absolute;
         top: -9999px;
         width: 100px;
         height: 100px;
         overflow-x: scroll;
     }
+`
+const hiddenScrollbarCSS = `
+    ${baseCSS}
     .${className}::-webkit-scrollbar {
         display: none;
     }
 `
 
-export function detectHorizontalScrollbarHeight() {
-    if (horizontalScrollbarHeight) {
-        return horizontalScrollbarHeight
+// This function assumes horizontal and vertical scrollbars
+// are always the same for a given browser/OS combo
+export function detectScrollbarSize(tryHiding) {
```

Comment:
I think it would be nice to have a jsdoc block explaining the `tryHiding` param, because the name doesn't tell me anything about it's purpose, I had to read the code (which wasn't complicated, but will be an extra effort, everything I'm scanning this file)

Also using verbs for non-function vars is not so good, I think. I assumed it's a function, so maybe something like `withHiddenScrollbar` is more appropriate